### PR TITLE
edits on TF page

### PR DESCRIPTION
**Pull request by LoryApiFortress**
### Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

File: docs/test-fairy/acct-mgmt/acct-settings.md [hunk 1]
```diff
@@ -8,132 +8,136 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+With the notifications options, you can define what type of messages you want to receive about new builds, crashes, and user feedback.
 
 <br/><img src={useBaseUrl('img/test-fairy/acc-settings/acc-set-4.png')} alt="Account Settings" width="600"/>
 
 ### Integrations
+
 You can integrate your TestFairy account with different services to customize and streamline your work processes.
 
-* SMTP and Gmail - See [SMTP and Gmail](https://docs.testfairy.com/Integrations/SMTP_and_Gmail.html) for information about connecting your SMTP email server or your Gmail account. This ensures the emails you send from your TestFairy account will be sent from the email account you used to register.
+- Simple Mail Transfer Protocol (SMTP) and Gmail - See [SMTP and Gmail](/test-fairy/integrations/smtp-gmail) for information about connecting your SMTP email server or Gmail account. This ensures the emails you send from your TestFairy account will be sent from the email account you used to register.
```
Comment (LoryApiFortress) [inline docs/test-fairy/acct-mgmt/acct-settings.md:39]:
```suggestion
- Simple Mail Transfer Protocol (SMTP) and Gmail - See [SMTP and Gmail](/test-fairy/integrations/smtp-gmail) for information about connecting your SMTP email server or Gmail account. This ensures you send the emails from the same account you used to register.
```

File: docs/test-fairy/acct-mgmt/acct-settings.md [hunk 2]
```diff
@@ -8,132 +8,136 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
-* ``[after adding SAML/Single Sign-on]`` (optional) - Grants access to all apps to testers who sign in with SSO.
+
+- **Require user login before app download** - If you want testers to first login to their tester account prior to downloading your app.
+- **Require [Google] Sign on for all users** - Users must sign in using a Google email address.
+- `[after adding SAML/Single Sign-on]` (optional) - Grants access to all apps to testers who sign in with SSO.
 
 #### SAML/Single Sign-On
-Here you can add the [SSO](https://docs.testfairy.com/Single_Sign-On/SSO.html) metadata definitions file. When you add SAML/Single Sign-On, the file will contain your ID, URL and x509 certificate.
+
+Add the [SSO](/test-fairy/acct-mgmt/sso/sso-intro) metadata definitions file here. When you add SAML/Single Sign-On, the file will contain your ID, URL, and x509 certificate.
```
Comment (LoryApiFortress) [inline docs/test-fairy/acct-mgmt/acct-settings.md:71]:
```suggestion
Add the [SSO](/test-fairy/acct-mgmt/sso/sso-intro) metadata definitions file here. When you add SAML/Single Sign-On, the file contains your ID, URL, and X.509 certificate.
```

File: docs/test-fairy/acct-mgmt/acct-settings.md [hunk 3]
```diff
@@ -8,132 +8,136 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+- Simple Mail Transfer Protocol (SMTP) and Gmail - See [SMTP and Gmail](/test-fairy/integrations/smtp-gmail) for information about connecting your SMTP email server or Gmail account. This ensures you send the emails from the same account you used to register.
 
-* Slack - See [Slack](https://docs.testfairy.com/Integrations/Slack.html) for information about integrating your Slack account with TestFairy.
+- Slack - See [Slack](/test-fairy/integrations/slack) for information about integrating your Slack account with TestFairy.
 
-* Zendesk - See [ZenDesk](https://docs.testfairy.com/Integrations/Zendesk.html) for information about integrating your Zendesk account with TestFairy.
+- Zendesk - See [ZenDesk](/test-fairy/integrations/zendesk) for information about integrating your Zendesk account with TestFairy.
 
-* Webhooks - You can use TeftFairy webhooks to connect to services. See [Microsoft Teams](https://docs.testfairy.com/Integrations/Microsoft_Teams.html) for more information.
+- Webhooks - You can use TestFairy webhooks to connect to services. See [Microsoft Teams](h/test-fairy/integrations/ms-teams) for more information.
```
Comment (LoryApiFortress) [inline docs/test-fairy/acct-mgmt/acct-settings.md:45]:
```suggestion
- Webhooks - You can use TestFairy webhooks to connect to services. See [Microsoft Teams](/test-fairy/integrations/ms-teams) for more information.
```

File: docs/test-fairy/acct-mgmt/acct-settings.md [hunk 4]
```diff
@@ -8,132 +8,136 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
 
 <br/><img src={useBaseUrl('img/test-fairy/acc-settings/acc-set-5.png')} alt="Account Settings" width="600"/>
 
 ### Bug Systems
-You can connect your bug system to TestFairy to report bugs directly to your [Jira Cloud](https://docs.testfairy.com/Bug_Tracking/JIRA_Cloud.html), [GitHub](https://docs.testfairy.com/Bug_Tracking/Github.html), or [Trello](https://docs.testfairy.com/Bug_Tracking/Trello.html) account.
+
+You can connect your bug system to TestFairy to report bugs directly to your [Jira Cloud](/test-fairy/testing-an-app/bug-tracking/jira-cloud), [GitHub](docs/test-fairy/testing-an-app/bug-tracking/github), or [Trello](docs/test-fairy/testing-an-app/bug-tracking/trello) account.
```
Comment (LoryApiFortress) [inline docs/test-fairy/acct-mgmt/acct-settings.md:51]:
```suggestion
You can connect your bug system to TestFairy to report bugs directly to your [Jira Cloud](/test-fairy/testing-an-app/bug-tracking/jira-cloud), [GitHub](/test-fairy/testing-an-app/bug-tracking/github), or [Trello](/test-fairy/testing-an-app/bug-tracking/trello) account.
```

File: docs/test-fairy/acct-mgmt/acct-settings.md [hunk 5]
```diff
@@ -8,132 +8,136 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
 <br/><img src={useBaseUrl('img/test-fairy/acc-settings/acc-set-1.png')} alt="Account Settings" width="600"/>
 
 The first two menu items are your SDK and API app tokens.
 
 ### SDK Token
+
 <br/><img src={useBaseUrl('img/test-fairy/acc-settings/acc-set-2.png')} alt="Account Settings" width="600"/>
 
-Your app token is used to initialize the [TestFairy SDK](https://docs.testfairy.com/SDK/Adding_The_Testfairy_SDK_To_Your_App.html).
+Your app token initializes the [TestFairy SDK](https://docs.testfairy.com/SDK/Adding_The_Testfairy_SDK_To_Your_App.html).
```
Comment (lysannep) [inline docs/test-fairy/acct-mgmt/acct-settings.md:21]: Should this link go /test-fairy/sdk/adding-tf-sdk/?

File: docs/test-fairy/acct-mgmt/acct-settings.md [hunk 6]
```diff
@@ -8,132 +8,136 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
 
-##Adding and Managing Users
+## Adding and Managing Users
 
 Use the Team page to manage your team members:
 
 <br/><img src={useBaseUrl('img/test-fairy/acc-settings/acc-set-12.png')} alt="Account Settings" width="600"/>
 
-To add admins to your account, go to the [Sauce Labs Team Management page](https://docs.saucelabs.com/basics/acct-team-mgmt-hub/).
+To add admins to your account, visit [Sauce Labs Team Management](https://docs.saucelabs.com/basics/acct-team-mgmt-hub/).
```
Comment (lysannep) [inline docs/test-fairy/acct-mgmt/acct-settings.md:105]:
```suggestion
To add admins to your account, see [Sauce Labs Team Management](/basics/acct-team-mgmt-hub).
```

File: docs/test-fairy/acct-mgmt/acct-settings.md [hunk 7]
```diff
@@ -8,132 +8,136 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
-At TestFairy we understand that at some point you may want to delete your data. As much as we are sad to see you leaving us, we will help you make it happen.
+At TestFairy, we understand that you may want to delete your data. We are sad to see you leaving us, but we will help you make it happen.
 
-First, this document is for __developers__, users who uploaded apps to TestFairy, added the TestFairy SDK to their app, and invited testers to their project. 
+First, this document is for **developers**, users who uploaded apps to TestFairy, added the TestFairy SDK to their app, and invited testers to their project.
 If you are a developer, please continue reading.
 
+If you are a **tester**, get in touch with the developer who invited you to their project and ask to be removed.
+
+#### How to Delete Your Developer Account
```
Comment (lysannep) [inline docs/test-fairy/acct-mgmt/acct-settings.md:116]:
```suggestion
#### Delete Your Developer Account
```

File: docs/test-fairy/acct-mgmt/acct-settings.md [hunk 8]
```diff
@@ -8,132 +8,136 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+Allows recording of specific users instead of recording all user sessions. It helps in cases in which customer support teams want to record users who are having issues.
 
 <br/><img src={useBaseUrl('img/test-fairy/acc-settings/acc-set-11.png')} alt="Account Settings" width="600"/>
 
-##Adding and Managing Users
+## Adding and Managing Users
 
 Use the Team page to manage your team members:
 
 <br/><img src={useBaseUrl('img/test-fairy/acc-settings/acc-set-12.png')} alt="Account Settings" width="600"/>
```
Comment (lysannep) [inline docs/test-fairy/acct-mgmt/acct-settings.md:103]:
```suggestion
<br/><img src={useBaseUrl('img/test-fairy/acc-settings/acc-set-12.png')} alt="Account Settings" width="500"/>
```

**Review by lysannep — APPROVED** [review]

File: docs/test-fairy/acct-mgmt/acct-settings.md (hunk 5 above)
Comment (LoryApiFortress) [reply docs/test-fairy/acct-mgmt/acct-settings.md:21]: This page is not supported in sauce

**Pull request merged**